diabetes_regression/util/logger.py

```diff
@@ -0,0 +1,20 @@
+from logging import Formatter, getLogger, DEBUG, INFO, StreamHandler
+import logging.config
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+import os
+from util.logging_config import logging_config
+
+def setup():
+    logging.config.dictConfig(logging_config)
+    logger = getLogger()
+
+    azure_log_handler = AzureLogHandler(
+        instrumentation_key=os.environ.get("APPINSIGHTS_INSTRUMENTATION_KEY"),
```

Comment:
I wonder if we should check if this variable is present and only instantiate and add the handler if so. In theory, even if we don't add the handler, things should work well in logging, although not published obviously.

I'm thinking more from the point of running this locally and if we want to force developers to push logs to App Insights.

Reply:
Added the log_to_local only flag, default is `false` - log to app insight.

Reply:
Added `log_to_local_only`, which has a default value of `false`. Has to set this to true if running locally and don't want to log on App insight.